Models.py

```diff
@@ -195,6 +242,15 @@ def load(self) -> Optional[Self]:
         return self.model
 
     def update_cached_online(self):
+        """
+        This method will update the cached data online.
+
+        Args:
+            None
+
+        Returns:
+            None
```

Comment:
Delete.

Smush into """This method updates the cached data using the internet""""
or something like that